packages/vscode-lit-plugin/src/test/scripts/test-runner.ts

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env node
+
+// A script that launches vscode with our extension installed and
+// executes ./mocha-driver
+
+import * as path from "path";
+
+import { runTests } from "@vscode/test-electron";
+
+async function main() {
+	try {
+		// Passed to `--extensionDevelopmentPath`
+		const extensionDevelopmentPath = path.resolve(__dirname, "../../../");
+		// The path to the extension test runner script
+		const extensionTestsPath = path.resolve(__dirname, "./mocha-driver");
+
+		// Download VS Code, unzip it and run the integration test
+		await runTests({ extensionDevelopmentPath, extensionTestsPath });
```

Comment:
you might be able to use top-level await now?

Reply:
Would be mildly cool, but would need to update the tsconfig for the whole project, and it's another thing that could go wrong.